legacy/builder/print_used_and_not_used_libraries.go

```diff
@@ -47,10 +47,10 @@ func (s *PrintUsedAndNotUsedLibraries) Run(ctx *types.Context) error {
 		if len(libResResult.NotUsedLibraries) == 0 {
 			continue
 		}
-		logger.Fprintln(os.Stdout, logLevel, constants.MSG_LIBRARIES_MULTIPLE_LIBS_FOUND_FOR, header)
-		logger.Fprintln(os.Stdout, logLevel, constants.MSG_LIBRARIES_USED, libResResult.Library.InstallDir)
+		logger.Fprintln(os.Stdout, logLevel, tr("Multiple libraries were found for \"{0}\""), header)
+		logger.Fprintln(os.Stdout, logLevel, " "+tr("Used: {0}"), libResResult.Library.InstallDir)
```

Comment:
I'd keep the space in the string, for right to left languages it should be on the other side I guess?

```suggestion
		logger.Fprintln(os.Stdout, logLevel, tr(" Used: {0}"), libResResult.Library.InstallDir)
```

Reply:
My main concern here is that the translator will not even notice it and will just translate the string dropping the space. This is further confirmed by looking at the Transifex GUI (in this case on the Italian language but I bet that it's the same for many others):

![image](https://user-images.githubusercontent.com/423515/132238272-f879567c-1e01-4e2d-a55c-059bac817b05.png)

as you can see the translated string has lost the whitespace.
IMHO we should keep formatting tricks out of the translated strings and add the space on the left or the right depending on the locale. But this implies that we need to handle the right-to-left support in a more profound way.

Reply:
Yeah, I agree. Probably we should try check the [golang/x/text](https://pkg.go.dev/golang.org/x/text) package and see if it helps.

All in all we should also rework how we output in the CLI.